pom.xml

```diff
@@ -323,6 +327,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
There is also a confluent pom present where the dependencies should be added to. Please, check [this pr](https://github.com/snowflakedb/snowflake-kafka-connector/pull/800) as an example.

Reply:
ahh, i missed that as it does not have maven icon in intellij :D thanks!